src/app.module.ts

```diff
@@ -1,10 +1,12 @@
 import { Module } from '@nestjs/common';
 import { AppController } from './app.controller';
 import { AppService } from './app.service';
+import { CatsController } from './cats/cats.controller';
+import { CatService } from './cats/cats.service';
 
 @Module({
   imports: [],
```

Comment:
- separate out cats module, it allows us to group related items in a single module
- import the cats.module.ts here
- by doing this things related to cats are placed under cats folder and access to items in cats occurs through it